acm/src/main/java/org/literacybridge/acm/store/Committable.java

```diff
@@ -83,7 +82,11 @@ public final void commit(Transaction t) {
         }
     }
 
-    public abstract void doCommit(Transaction t) throws IOException;
+    /**
```

Comment:
Hmm I don't understand this question. There are three types of events: add, change, delete.